cmd/mtchannel_broker/main.go

```diff
@@ -19,14 +19,21 @@ package main
 import (
 	// Uncomment the following line to load the gcp plugin (only required to authenticate against GKE clusters).
 	// _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"flag"
 
 	"knative.dev/pkg/injection/sharedmain"
+	"knative.dev/pkg/signals"
 
 	"knative.dev/eventing/pkg/reconciler/mtbroker"
 )
 
 func main() {
-	sharedmain.Main("mt-broker-controller",
-		mtbroker.NewController,
-	)
+	clientQPS := flag.Float64("clientQPS", 5.0, "Overrides rest.Config.DefaultQPS.")
+	clientBurst := flag.Int("clientBurst", 10.0, "Overrides rest.Config.Burst.")
```

Comment:
```suggestion
	// TODO(https://github.com/knative/eventing/issues/3591): switch back to sharedmain.Main
	clientQPS := flag.Float64("clientQPS", rest.DefaultQPS, "Overrides rest.Config.QPS.")
	clientBurst := flag.Int("clientBurst", rest.DefaultBurst, "Overrides rest.Config.Burst.")
```

Reply:
I can do this, but I think it would still be useful to expose some of these knobs for tuning things?

Reply:
In the context here, we are exposing the knobs to raise the limits due to that linked issue, which is why I thought it most relevant here.

I agree the knobs are useful, but they probably belong in sharedmain, and ideally this would go back to being a fairly thin invocation of that.